src/features/community/scientist/ScientistModal.tsx

```diff
@@ -0,0 +1,69 @@
+import React, { useContext, useState } from "react";
+import { useActor, useMachine } from "@xstate/react";
+import { Modal } from "react-bootstrap";
+
+import { Panel } from "components/ui/Panel";
+import { Tab } from "components/ui/Tab";
+import * as AuthProvider from "features/auth/lib/Provider";
+import { Context } from "features/community/lib/CommunityProvider";
+import { incubateMachine } from "./lib/incubateMachine";
+
+// images
+import tadpole_icon from "../assets/icons/tadpole.png";
+import incubator from "../assets/incubator/algae-small.gif";
+import close from "assets/icons/close.png";
+
+import { InventoryItems } from "./components/InventoryItems";
+import { Incubator } from "./components/Incubator";
+
+interface Props {
+  isOpen: boolean;
+  onClose: () => void;
+}
+
+export const ScientistModal: React.FC<Props> = ({ isOpen, onClose }) => {
+  const { authService } = useContext(AuthProvider.Context);
+  const [authState] = useActor(authService);
+  const { communityService } = useContext(Context);
+  const [communityState] = useActor(communityService);
+  const [machine, send] = useMachine(incubateMachine);
+  const { state, errorCode } = machine.context;
+
+  const handleClose = () => onClose();
+
+  const [tab, setTab] = useState<"inventory" | "incubator">("inventory");
+
+  return (
+    <Modal centered show={isOpen} onHide={handleClose}>
+      <Panel className="relative max-w-5xl">
```

Comment:
<img width="558" alt="Screen Shot 2022-11-03 at 2 04 30 pm" src="https://user-images.githubusercontent.com/11745561/199640244-5768edc0-ce5a-460e-99aa-bbfe8e5d2184.png">

For the 'tab' style you will need to put some padding at the top of your Panel. See attached for how tabs should appear.